internal/build.go

```diff
@@ -117,6 +118,9 @@ func prepareIntermediate(ctx context.Context, opts *BuildOptions, schemaDataFunc
 }
 
 func finalizeEnvelope(ctx context.Context, env *gobl.Envelope, opts *BuildOptions) error {
+	if opts.Draft {
+		env.Head.Draft = true
+	}
```

Comment:
What if `opts.Draft` is false, and `env.Head.Draft` already is set to true? Should we unset it here?

Reply:
I don't think so. If the `--draft` flag is not present, we assume it should not be considered. A `--not-draft` could override this option, but I don't think it's needed.